data/navy watcher subplot

```diff
@@ -0,0 +1,673 @@
+					decline
+
+mission "Expedition to Watcher 2"
+	landing
+	name "Expedition to Watcher"
+	description "Transport the scientists and their cargo of biological specimens back home to <planet>."
+	source Watcher
+	destination Hermes
+	to offer
+		has "Expedition to Watcher 1: done"
+	passengers 4
+	cargo "biological specimens" 10
+	blocked "You have reached <origin>, but you need <capacity> for the scientists' cargo. Return here when you have the required space free."
+
+	on offer
+		event "watching watcher"
+		conversation
+			`You land on a relatively flat patch of land and help the scientists load your ship with all sorts of plants, many of which are both very large and surprisingly delicate. These plants could obviously never support their own weight on most other planets. Even the tallest trees here bend in the wind like the grass did back on New Boston.`
+			`	As you begin readying your ship to return to orbit, your radar picks up the signal of several other ships taking off from the tiny moon's surface. Their signatures are easy to read: pirates.`
+				launch
+	npc
+		personality nemesis timid plunders
+		government Pirate
+		fleet "Drug Runners"
+	npc
+		personality nemesis timid plunders
+		government Pirate
+		fleet "Drug Runners"
+	npc
+		personality nemesis timid plunders
+		government Pirate
+		fleet "Drug Runners"
+
+	on complete
+		payment 32000
+		dialog `The scientists thank you for bringing them to Watcher and pay you <payment>. Although many of the specimens have already been crushed by the forces of gravity, they assure you that they had fully expected this outcome.`
+
+mission "Poachers 1"
+	name "Poachers on Watcher"
+	description "Pick up Frank Moore on <destination>."
+	minor
+	source
+		near Capella 2 5
+		government "Republic"
+	destination Hermes
+	to offer
+		has "Expedition to Watcher 2: done"
+		random < 40
+	to fail
+		has "chosen sides"
+	cargo "research equipment" 3
+
+	on offer
+		conversation
+			`As you enjoy your drink at the spaceport bar, a Naval officer in his mid-to-late thirties approaches you. He looks distinctly familiar, though you can't figure out why. "Are you Captain <last>? I was told I might find you here." Without waiting for a reply, he sits down next to you. "Commander Henry Moore. My brother Frank is a research scientist working out on Hermes, and you recently brought him..."`
+			`	"To Watcher, yeah. I thought you reminded me of someone. You're brothers?"`
+			`	"Yes, Frank's my little brother." He smiles nervously.`
+			`	"So how can I help you?"`
+			`	"My brother... wants to hire you for another trip out to Watcher, to set up some... equipment." He's clearly hiding something. Poorly.`
+			choice
+				`	"Why didn't he contact me directly?"`
+					goto question
+				`	"Spit it out. What aren't you telling me?"`
+					goto question
+				`	"Okay, where do I pick him up?"`
+					goto equip
+				`	"Sorry, I'm not heading back toward Hermes any time soon."`
+					decline
```

Comment:
It's just that "sorry, I'm not heading back towards Hermes anytime soon" sounds more like a defer than a no. Maybe make the reply something along the lines of "sorry, I'm not interested"